src/com/android/settings/sudconfig/SudConfigProvider.kt

```diff
@@ -26,15 +27,19 @@ class SudConfigProvider : NonRelationalProvider() {
         )
     }
 
+    private lateinit var defaultThemeString: String
+
     override fun onCreate(): Boolean {
+        defaultThemeString = SystemProperties.get("setupwizard.theme")
+        if (TextUtils.isEmpty(defaultThemeString)) defaultThemeString = "glif_v4_light"
```

Comment:
`setupwizard.theme` should never be empty on GrapheneOS after https://github.com/GrapheneOS/platform_build/pull/56, it's better to throw an IllegalStateException here